tests/test_namespaces.py

```diff
@@ -39,32 +47,43 @@ def custom_namespaces(tmp_path):
 def generated_sql_uri(tmp_path):
     dest = tmp_path / "bigquery_etl.tar.gz"
     with tarfile.open(dest, "w:gz") as tar:
+
+        def add_to_tar(path, content):
+            content = dedent(content).lstrip()
+            info = tarfile.TarInfo(path)
+            info.size = len(content)
+            tar.addfile(info, BytesIO(content.encode()))
+
         for dataset in ("glean_app", "glean_app_beta"):
-            content = dedent(
-                f"""
+            content = f"""
                 references:
                   view.sql:
                   - moz-fx-data-shared-prod.{dataset}_derived.baseline_clients_daily_v1
                 """
-            ).lstrip()
```

Comment:
why is `dedent` removed here?

Reply:
I moved it to the function: https://github.com/mozilla/lookml-generator/pull/29/files#diff-771694b87a3d9b5cede55055ad91f9c22fc4fac470dd326fc6112ae662cc78acR52